README.md

```diff
@@ -56,6 +56,10 @@ You can also use the `--serve` option to `build.sh` to automatically serve the r
 
 Now you're ready to edit the `html/source` file—and after you make your changes, you can run the `build.sh` script again to see the new output.
 
+## Fast local iteration
+
+There are a number of options to disable certain parts of the build process to speed up local iteration. Run `./build.sh help` to see them all, or just use the `--fast` flag to get maximally-fast builds.
```

Comment:
I wasn't sure how much documentation we should have here, since it kind of duplicates the help output. Thoughts welcome.

Reply:
> I wasn't sure how much documentation we should have here, since it kind of duplicates the help output.

What you have looks sufficiently fine to me